docs/ecu-ElmProtocolTelegramProtocol.js

```
@@ -0,0 +1,286 @@
+        }
+
+        return false;
+
+      default:
+        if (buffer.charAt(0) === '+') {
+          return false;
+        }
+
+        this.lastRxMsg = buffer;
+
+        const firstChar = buffer.charAt(0);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)